src/client/app/translations/data.js

```diff
@@ -2,8 +2,6 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { localeData } from 'moment';
```

Comment:
This import isn't used as the localData declared on line 10 shadows it. Remove it as it could be a source of confusion.